site/content/blogs/release-v119.en.md

```diff
@@ -0,0 +1,97 @@
+* **`port` for Target Group Health Checks**:
+With the new `port` field, you can configure a non-default port for health checks, one different than that for requests from the load balancer.
+```yaml
+http:
+  path: '/'
+  healthcheck:
+    port: 8080
+```
+
+* **Bug fixes:** 
+    * Preserve tags applied by `app init --resource-tags` when services are deleted from an application ([#3582](https://github.com/aws/copilot-cli/pull/3582))
+    * Fix regression when enabling autoscaling fields for Load Balanced Web Services with Network Load Balancers ([#3578](https://github.com/aws/copilot-cli/pull/3578))
+    * Enable `copilot svc exec` for Fargate Windows tasks ([#3566](https://github.com/aws/copilot-cli/pull/3566))
+
+There is no breaking change in this release.
+
+## What’s AWS Copilot?
+
+The AWS Copilot CLI is a tool for developers to build, release, and operate production ready containerized applications on AWS.
+From getting started, pushing to staging, and releasing to production, Copilot can help manage the entire lifecycle of your application development.
+At the foundation of Copilot is AWS CloudFormation, which enables you to provision infrastructure as code in a single operation.
+Copilot provides pre-defined CloudFormation templates and user-friendly workflows for different types of micro services creation and operation,
+enabling you to focus on developing your application, instead of writing deployment scripts.
+
+See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
+
+## Internal Load Balancers
+_Contributed by [Janice Huang](https://github.com/huanjani) and [Danny Randall](https://github.com/dannyrandall)_  
+By configuring a few things when you initiate your Copilot environment and workload, you can now create an [internal load balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-internal-load-balancers.html), whose nodes have only private IP addresses.
+
+The internal load balancer is an environment-level resource, to be shared among other permitted services. When you run `copilot env init`, you can import some specific resources to support the ALB. For services with `https` capability, use the [`--import-cert-arns`](../docs/commands/env-init.en.md#what-are-the-flags) flag to import the ARNs of your existing private certificates. If you'd like your ALB to receive ingress traffic within the environment VPC, use the [`--internal-alb-allow-vpc-ingress`](../docs/commands/env-init.en.md#what-are-the-flags) flag; otherwise, by default, access to the internal ALB will be limited to only Copilot-created services within the environment. For now, Copilot will associate imported certs with an internal ALB *only if* the environment's VPC has no public subnets, so import only private subnets.
```

Comment:
```suggestion
The internal load balancer is an environment-level resource, to be shared among other permitted services. When you run `copilot env init`, you can import some specific resources to support the ALB. For services with `https` capability, use the [`--import-cert-arns`](../docs/commands/env-init.en.md#what-are-the-flags) flag to import the ARNs of your existing private certificates.  For now, Copilot will associate imported certs with an internal ALB *only if* the environment's VPC has no public subnets, so import only private subnets. If you'd like your ALB to receive ingress traffic within the environment VPC, use the [`--internal-alb-allow-vpc-ingress`](../docs/commands/env-init.en.md#what-are-the-flags) flag; otherwise, by default, access to the internal ALB will be limited to only Copilot-created services within the environment.
```